tests/testing_utils/realtime/test_realtime.py

```diff
@@ -0,0 +1,118 @@
+"""
+Test module for the realtime code.
+Run via: 'python3 test_realtime.py'
+:copyright: 2022 SuperDARN Canada
+:author: Kevin Krieger
+"""
+import unittest
+import os
+import sys
+
+if not os.environ['BOREALISPATH']:
+    BOREALISPATH = f"{os.environ['HOME']}/PycharmProjects/borealis/"
+else:
+    BOREALISPATH = os.environ['BOREALISPATH']
+
+sys.path.append(BOREALISPATH)
+
+from src import realtime
```

Comment:
```suggestion
from src.realtime import RealtimeServer
```

Reply:
You would need to get a logger started before this. You can also disable logs from imported modules.